### ISSUE TITLE
Features page

### ISSUE THREAD

**Issue by sumobi** [pippinsplugins/rcp-site-child#26]
It would be great to have 1 page where someone can see every single feature included, similar to the AffiliateWP one, only prettier http://affiliatewp.com/features/.

**Comment by pippinsplugins** [comment]
Yes.

**Comment by sumobi** [comment]
@pippinsplugins I'm thinking of doing this in the same way as the payment gateways page #35. 1 page template set on the parent "features" page. The sub pages are the features, but you can't click through to each one. The content and featured image will simply be shown on the features page.

**Comment by pippinsplugins** [comment]
Yes. I love that idea. That way we can can summarize each feature and then elaborate on it in depth.

**Comment by sumobi** [comment]
@pippinsplugins :+1: I was actually thinking each feature would not link through to a page by itself, but it could be an option if there was a lot of content (or a lightbox?).

**Comment by pippinsplugins** [comment]
Maybe a checkbox to enable / disable the link?
